hw/dv/data/dsim/dsim.hjson

```diff
@@ -81,7 +80,7 @@
   ]
 
   // Defaults for DSim
-  cov_metrics:          ""
+  cov_metrics:          "metrics.db"
```

Comment:
This option is meant to indicate what coverage metrics to collect (line, cond, expr etc). By default this needs to be set to DSim's equivalent of 'all'. The `cov_db_dir` is used to set the path to the coverage database (directory).

Reply:
Thanks for the clarification. DSim currently doesn't have an equivalent of "all" coverage metrics, although I anticipate that will change in the near future. Since the command line arguments used to control code coverage are in flux right now, do you mind if I leave this blank? I could leave a TODO comment in place and address this in a future pull request.

Let me know if you need to use code coverage immediately, and I will supply the currently implemented compile and run time arguments. Thanks!

Reply:
Sure, you could leave it blank for now.